### PR TITLE
Update file - /courses/language/ptbr

### PR DESCRIPTION
**Pull request by calixtocarolina**
Update file with course in PTBR in the cybersecurity field.

## What does this PR do?
Add info & Improve repo

### Description

I'm adding new courses in the cybersecurity field (ptbr language), totally free and some with certificate guaranted to the students. 

### Why is this valuable (or not)?

It is valuable because in this field (cybersecurity), it's hard to find free resources, books and courses to study and improve.

### How do we know it's really free?

Before I submitted, I checked all the links.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by metabiswadeep** [comment]
@calixtocarolina Write the name of the books in alphabetical order and not randomly to avoid linter error.

**Comment by calixtocarolina** [comment]
> @calixtocarolina Write the name of the books in alphabetical order and not randomly to avoid linter error.

Hello! I'm sorry, but I didn't referenced any books. My contribution was adding new courses. How do I fix it?

**Review by calixtocarolina — COMMENTED** [review]
(fix): add courses in PTBR in alphabetical order

**Review by metabiswadeep — CHANGES_REQUESTED** [review]
Order should be:
* [Boas Práticas de Segurança da Informação para Sua Empresa](https://www.udemy.com/course/empresa-mais-segura/) - Afonso da Silva E. (Udemy)
* [Introdução ao Hacking e Pentest](https://solyd.com.br/treinamentos/introducao-ao-hacking-e-pentest-2/) - Guilherme Junqueira (Solyd)
* [Proteção de Dados](https://educacao-executiva.fgv.br/cursos/online/curta-media-duracao-online/protecao-de-dados?gclid=Cj0KCQjw06OTBhC_ARIsAAU1yOUI11DW2VnmsFESfaI9mZ3Kb2KiKSZZ6hsP3fCxONGiFssgjqGNzdsaAmJyEALw_wcB) - FGV - Proteção de Dados
* [Segurança da Informação: Por onde iniciar sua carreira](https://www.udemy.com/course/seguranca-da-informacao-por-onde-iniciar-sua-carreira/) - Alexandro Silva (Udemy)
* [Splunk - Fundamentos](https://www.youtube.com/watch?v=AIJKMk8kunw&list=PL8vOoYAs_ySzhFsDzfM-_JbEfMJtmtdVp) - Splunk - Fundamentos (Youtube)

File: courses/free-courses-pt_BR.md [hunk 1]
```diff
@@ -354,3 +354,6 @@
 
 * [Boas Práticas de Segurança da Informação para Sua Empresa](https://www.udemy.com/course/empresa-mais-segura/) - Afonso da Silva E. (Udemy)
 * [Segurança da Informação: Por onde iniciar sua carreira](https://www.udemy.com/course/seguranca-da-informacao-por-onde-iniciar-sua-carreira/) - Alexandro Silva (Udemy)
+* [Proteção de Dados](https://educacao-executiva.fgv.br/cursos/online/curta-media-duracao-online/protecao-de-dados?gclid=Cj0KCQjw06OTBhC_ARIsAAU1yOUI11DW2VnmsFESfaI9mZ3Kb2KiKSZZ6hsP3fCxONGiFssgjqGNzdsaAmJyEALw_wcB) - FGV - Proteção de Dados
+* [Introdução ao Hacking e Pentest](https://solyd.com.br/treinamentos/introducao-ao-hacking-e-pentest-2/) - Guilherme Junqueira (Solyd)
+* [Splunk - Fundamentos](https://www.youtube.com/watch?v=AIJKMk8kunw&list=PL8vOoYAs_ySzhFsDzfM-_JbEfMJtmtdVp) - Splunk - Fundamentos (Youtube)
```
Comment (metabiswadeep) [inline courses/free-courses-pt_BR.md:359]:
```suggestion
* [Boas Práticas de Segurança da Informação para Sua Empresa](https://www.udemy.com/course/empresa-mais-segura/) - Afonso da Silva E. (Udemy)
* [Introdução ao Hacking e Pentest](https://solyd.com.br/treinamentos/introducao-ao-hacking-e-pentest-2/) - Guilherme Junqueira (Solyd)
* [Proteção de Dados](https://educacao-executiva.fgv.br/cursos/online/curta-media-duracao-online/protecao-de-dados?gclid=Cj0KCQjw06OTBhC_ARIsAAU1yOUI11DW2VnmsFESfaI9mZ3Kb2KiKSZZ6hsP3fCxONGiFssgjqGNzdsaAmJyEALw_wcB) - FGV - Proteção de Dados
* [Segurança da Informação: Por onde iniciar sua carreira](https://www.udemy.com/course/seguranca-da-informacao-por-onde-iniciar-sua-carreira/) - Alexandro Silva (Udemy)
* [Splunk - Fundamentos](https://www.youtube.com/watch?v=AIJKMk8kunw&list=PL8vOoYAs_ySzhFsDzfM-_JbEfMJtmtdVp) - Splunk - Fundamentos (Youtube)
```

**Review by metabiswadeep — CHANGES_REQUESTED** [review]

File: courses/free-courses-pt_BR.md (hunk 1 above)
Comment (eshellman) [inline courses/free-courses-pt_BR.md:359]: For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

File: courses/free-courses-pt_BR.md [hunk 2]
```diff
@@ -354,3 +354,6 @@
 
 * [Boas Práticas de Segurança da Informação para Sua Empresa](https://www.udemy.com/course/empresa-mais-segura/) - Afonso da Silva E. (Udemy)
 * [Segurança da Informação: Por onde iniciar sua carreira](https://www.udemy.com/course/seguranca-da-informacao-por-onde-iniciar-sua-carreira/) - Alexandro Silva (Udemy)
+* [Proteção de Dados](https://educacao-executiva.fgv.br/cursos/online/curta-media-duracao-online/protecao-de-dados?gclid=Cj0KCQjw06OTBhC_ARIsAAU1yOUI11DW2VnmsFESfaI9mZ3Kb2KiKSZZ6hsP3fCxONGiFssgjqGNzdsaAmJyEALw_wcB) - FGV - Proteção de Dados
+* [Introdução ao Hacking e Pentest](https://solyd.com.br/treinamentos/introducao-ao-hacking-e-pentest-2/) - Guilherme Junqueira (Solyd)
```
Comment (eshellman) [inline courses/free-courses-pt_BR.md:358]: seems there is a second author

File: courses/free-courses-pt_BR.md [hunk 3]
```diff
@@ -354,3 +354,6 @@
 
 * [Boas Práticas de Segurança da Informação para Sua Empresa](https://www.udemy.com/course/empresa-mais-segura/) - Afonso da Silva E. (Udemy)
 * [Segurança da Informação: Por onde iniciar sua carreira](https://www.udemy.com/course/seguranca-da-informacao-por-onde-iniciar-sua-carreira/) - Alexandro Silva (Udemy)
+* [Proteção de Dados](https://educacao-executiva.fgv.br/cursos/online/curta-media-duracao-online/protecao-de-dados?gclid=Cj0KCQjw06OTBhC_ARIsAAU1yOUI11DW2VnmsFESfaI9mZ3Kb2KiKSZZ6hsP3fCxONGiFssgjqGNzdsaAmJyEALw_wcB) - FGV - Proteção de Dados
```
Comment (eshellman) [inline courses/free-courses-pt_BR.md:357]: not familiar with this platform - how do you know it's free

**Pull request closed without merging**